### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by adeeena**
Waiting for peer review whether everything is well translated.

FYI I put my name in the index.md, in a blockquote, same level as the ToC. I'll do thr same with the other translation I made once this PR gets accepted.

### CODE REVIEW

**Comment by ko-pp** [comment]
Hey, thanks for translating in French <3

I wanted to point out that the Gender Unicorn image is available in French: https://transstudent.org/wp-content/uploads/2017/08/genderunicornfran.jpg :slightly_smiling_face:

**Comment by adeeena** [comment]
Nice, thank you!

File: public/fr/_concat.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "public/fr/impostor-syndrome.md",
+    "public/fr/am-i-trans.md",
+    "public/fr/diagnoses.md",
+    "public/fr/treatment.md",
+    "public/fr/causes.md",
+    "public/fr/chromosomes.md",
+    "public/fr/hormones.md",
+    "public/fr/second-puberty-masc.md",
+    "public/fr/second-puberty-fem.md",
+    "public/fr/conclusion.md"
```
Comment (Twipped) [inline public/fr/_concat.json:25]: These filenames are all wrong, but I can correct it locally.

**Comment by adeeena** [comment]
Hello, I am still unable to access to the french translation on the public site, is there a planned update on it?

**Pull request merged**